gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/inspection/inspect.ts

```diff
@@ -136,6 +177,37 @@ export default class OCR2Inspect extends SolanaCommand {
     )
 
     const onChainOCRConfig = deserializeConfig(bufferedConfig)
+
+    // Print on-chain oracle information
+    onChainState.oracles.xs.forEach((oracle) => {
+      logger.info(
+        `Oracle Info:
+          - Transmitter: ${oracle.transmitter}
+          - Proposed Payee: ${oracle.proposedPayee}
+          - From Round ID: ${oracle.fromRoundId}
+          - Payment Gjuels: ${oracle.paymentGjuels}
+      `,
+      )
+    })
+
+    const input = this.makeInput(this.flags.input)
+    // If input does not exist, just print config
+    if (!input.description) {
+      logger.info(`Min Answer: ${onChainState.config.minAnswer}`)
```

Comment:
Just few lines before we group all logs together:

```javascript
logger.info(
        `Oracle Info:
          - Transmitter: ${oracle.transmitter}
          - Proposed Payee: ${oracle.proposedPayee}
          - From Round ID: ${oracle.fromRoundId}
          - Payment Gjuels: ${oracle.paymentGjuels}
      `,
      )
```

We should define a standard way to log large data objects like this, so we get a unified look & feel at the output.